init_joints:
  joint_1: 0.7726
  joint_2: -0.00448182
  joint_3: -0.84231
  joint_4: -1.77722
  joint_5: -0.0103137
  joint_6: 1.62066
  joint_7: 0.703299
